Assets/Python/Screens/CvCorporationScreen.py

```diff
@@ -102,6 +102,7 @@ def __init__(self):
 		self.W_INFLUENCES_AREA = self.X_CITY2_AREA + self.W_CITY_AREA - self.X_INFLUENCES_AREA
 		self.H_INFLUENCES_AREA = self.H_REQUIREMENTS_AREA
 		self.INFLUENCES_ID =  "InfluencessAreaWidget"
+		self.iCorporationTech = iCorporation

```

Comment:
I suggest to solve this using
```
from Consts import *
from Consts import iCorporation as iCorporationTech
```

Should work iirc. At least the iCorporationTech shouldn't be a member variable of the CvCorporationScreen class.